### PR TITLE
Add runbook

### PR DESCRIPTION
**Pull request by mchompalova**
# Description

## What

Add runbook for the service.

## Why

https://financialtimes.atlassian.net/browse/UPPSF-1181

## Scope and particulars of this PR (Please tick all that apply)

- [ ] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [x] Feature
- [ ] Documentation
- [ ] Breaking change
- [ ] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

### CODE REVIEW

File: runbooks/runbook.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+No
+
+## Contains Sensitive Data
+
+No
+
+## Dependencies
+
+* next-searchapi
```
Comment (karsov) [inline runbooks/runbook.md:58]: This definitely is not a dependency.

Comment (mchompalova) [reply runbooks/runbook.md:58]: It was `search-api` but the runbook validation failed.

File: runbooks/runbook.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+* kafka-proxy
+* public-concordances-api
+* up-ica
```
Comment (karsov) [inline runbooks/runbook.md:61]: https://biz-ops.in.ft.com/System/upp-content-es-cluster is a dependency too.

File: runbooks/runbook.md (hunk 1 above)
Comment (mchompalova) [reply runbooks/runbook.md]: I fixed it.

File: runbooks/runbook.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+- robert.marinov
+- tsvetan.dimitrov
+
+## Host Platform
+
+AWS
+
+## Architecture
+
+This service receives combined messages (content and metadata) from the `CombinedPostPublicationEvents` Kafka queue and indexes them in Elasticsearch for use by SAPI v1.
```
Comment (karsov) [inline runbooks/runbook.md:46]: While adding a separate system for the "reindexer" deployment maybe an overkill, we may add here that the "content-rw-elasticsearch-reindexer" listens to the "ForcedCombinedPostPublicationEvents" queue.

Comment (mchompalova) [reply runbooks/runbook.md:46]: Let's leave it for the next PR :)

**Pull request merged**